.github/workflows/publish-page.yml

```
@@ -24,10 +24,14 @@ jobs:
     steps:
       - name: Checkout
         uses: actions/checkout@v4
-      
+      - name: Install pnpm
+        uses: pnpm/action-setup@v3
+        with:
+          version: 9
       - uses: actions/setup-node@v4
         with:
-          node-version: 16
+          node-version: 20
+          cache: 'pnpm'
```

probably needs `cache-dependency-path: 'docs'` as well to work
```suggestion
          cache: 'pnpm'
          cache-dependency-path: 'docs'
```